### PR TITLE
Handle mode change

### PR DESCRIPTION
**Pull request by ppapadeas**

### CODE REVIEW

**Comment by ppapadeas** [comment]
@johngian r?

File: satnogsclient/web/app.py [hunk 1]
```diff
@@ -99,19 +99,45 @@ def get_command():
...
                 # TODO: Handle the comms_tx_rf request
                 if requested_command['custom_cmd']['comms_tx_rf'] == 'comms_off':
                     packet.comms_off()
                     response['log_message'] = 'COMMS_OFF command sent'
                     response['id'] = 1
+                    return jsonify(response)
                 elif requested_command['custom_cmd']['comms_tx_rf'] == 'comms_on':
                     packet.comms_on()
                     response['log_message'] = 'COMMS_ON command sent'
                     response['id'] = 1
```
Comment (johngian) [inline satnogsclient/web/app.py:113]: `response['id']` is the same in both if/elif cases. Please move it outside.

File: satnogsclient/web/app.py [hunk 2]
```diff
@@ -99,19 +99,45 @@ def get_command():
...
-                return jsonify(response)
+                    return jsonify(response)
+            elif 'mode' in requested_command['custom_cmd']:
+                # TODO: Handle the comms_tx_rf request
+                mode = requested_command['custom_cmd']['mode']
+                if requested_command['custom_cmd']['mode'] == 'cmd_ctrl':
+                    response['log_message'] = 'Mode changed to Stand-Alone'
+                    response['id'] = 1
+                    # return jsonify(response)
```
Comment (johngian) [inline satnogsclient/web/app.py:121]: No need to keep these commented out lines.

File: satnogsclient/web/app.py [hunk 3]
```diff
@@ -99,19 +99,45 @@ def get_command():
...
+                elif requested_command['custom_cmd']['mode'] == 'network':
+                    response['log_message'] = 'Mode changed to Network'
+                    response['id'] = 1
+                    # return jsonify(response)
```
Comment (johngian) [inline satnogsclient/web/app.py:125]: Same here.

File: satnogsclient/web/app.py [hunk 4]
```diff
@@ -99,19 +99,45 @@ def get_command():
...
+                dict_out = {'mode': mode}
+                packet.custom_cmd_to_backend(dict_out)
+            elif 'backend' in requested_command['custom_cmd']:
+                # TODO: Handle the comms_tx_rf request
+                backend = requested_command['custom_cmd']['backend']
+                if requested_command['custom_cmd']['backend'] == 'gnuradio':
+                    response['log_message'] = 'Backend changed to GNURadio'
+                    response['id'] = 1
+                    # return jsonify(response)
```
Comment (johngian) [inline satnogsclient/web/app.py:134]: Same here.

File: satnogsclient/web/app.py [hunk 5]
```diff
@@ -99,19 +99,45 @@ def get_command():
...
+                elif requested_command['custom_cmd']['backend'] == 'serial':
+                    response['log_message'] = 'Backend changed to Serial'
+                    response['id'] = 1
+                    # return jsonify(response)
```
Comment (johngian) [inline satnogsclient/web/app.py:138]: Same here.

**Pull request merged**